### ISSUE TITLE
Undocumented property false-positive

### ISSUE THREAD

**Issue by Koriit** [arturbosch/detekt#2599]
## Steps to Reproduce
```kotlin
class MyClass(
    /** My property documentation. */
    val property: String
)
```
## Observed Behavior
Detekt finds it to be undocumented property. Note that IDEA has no issue with such documentation placement:
![image](https://user-images.githubusercontent.com/8916393/79109346-74f54500-7d78-11ea-8868-d6543a476cd7.png)

Similar issue was brought up in #2529. Using `@param` and `@property` works fine - which I'm going to use for now, might even stick with it.

## Expected Behavior
I think this is false-positive and there should be no `UndocumentedPublicProperty`.

## Context
I have update detekt from 1.5.1 to 1.7.4 in my project. It's nice that it now detects undocumented public properties, however, it doesn't allow me to document them how I would like to.

## Your Environment
* Version of detekt used: `1.7.4`
* Version of Gradle used (if applicable): `6.3`
* Operating System and version: `macOS Mojave 10.14.6`
* Link to your project (if it's a public repository): https://github.com/Koriit/ktor-controllers

Thanks!

**Comment by BraisGabin** [comment]
This was fixed in #2591. It'll be released in the next version of detekt. Thanks for the report!

**Comment by Koriit** [comment]
Ok, thanks!
